.github/workflows/shippable_builds.yml

```diff
@@ -836,6 +870,10 @@ jobs:
               await core.summary.addRaw(`Released to the <b>${process.env.playTargetTrack}</b> track on Google Play`, true).write();
             }
 
+            if (process.env.skipFtpMo != "true" && process.env.releaseTarget.includes("ftpmo")) {
+              await core.summary.addRaw(`Published to ${process.env.FTPMO_URL_PREFIX}${process.env.ftpReleaseUrl}`, true).write();
```

Comment:
Are you going to use FTPMO_URL_PREFIX in other places? Otherwise I'd just go ahead and use the URL here directly.